core/src/main/java/org/apache/calcite/sql/dialect/BigQuerySqlDialect.java

```diff
@@ -2073,4 +2081,26 @@ private void unparseBitNotFunction(SqlWriter writer, SqlCall call) {
     call.operand(0).unparse(writer, 0, 0);
     writer.print(")");
   }
+
+  public void unparseRegexpExtractAllForStrtok(SqlWriter writer, SqlCall call,
+      int leftPrec, int rightPrec) {
+    SqlWriter.Frame regexpExtractAllFrame = writer.startFunCall("REGEXP_EXTRACT_ALL");
```

Comment:
Here we are not able to create SqlCall for REGEXP_EXTRACT_ALL because many diff patterns are directly used such as 'r' and the specific regex pattern.